### PR TITLE
Add date picker calendar to UIComponents sample

### PR DESCRIPTION
**Pull request by DohyungLim**
- Add Date picker Dialog (Calender view)
- Add Date picker (Calendar view)

### CODE REVIEW

File: examples/mobile/UIComponents/components/dialogs/pickers/calendar.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+    <link href="../../../css/style.css" rel="stylesheet" />
```
Comment (singa2000) [inline examples/mobile/UIComponents/components/dialogs/pickers/calendar.html]: Would you check indent, please?

Comment (DohyungLim) [reply examples/mobile/UIComponents/components/dialogs/pickers/calendar.html]: done.

File: examples/mobile/UIComponents/components/dialogs/pickers/calendar.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+</head>
+
+<body>
+	<div class="ui-page" data-role="page">
+		<div class="ui-header" data-position="fixed">
+			<h1>
+				Calendar
+			</h1>
+		</div>
+		<div class="ui-content">
```
Comment (singa2000) [inline examples/mobile/UIComponents/components/dialogs/pickers/calendar.html:19]: In the Guide, it is included in the List.
So the calendar width is same with List width.
Would you make the sample with List, please?

Comment (DohyungLim) [reply examples/mobile/UIComponents/components/dialogs/pickers/calendar.html:19]: done.

**Review by singa2000 — APPROVED** [review]
It's good.

**Pull request merged**